### PR TITLE
London Class 8-Hatice Aydogan-JavaScript-Core-1-Coursework-Week 1

### PR DESCRIPTION
**Pull request by Hatice06**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: extra/2-piping.js [hunk 1]
```diff
@@ -16,26 +16,35 @@
...
 }
 
+
 const startingValue = 2;
+  
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = format(multiply(add(startingValue, 10), 2));
+// MY COMMENT: This practice is not good as everything is in one line and hard to understand.
```
Comment (ewintram) [inline extra/2-piping.js:40]: I agree! It is still good code in that you are using named functions to communicate what is happening. An example of really bad code would be to not use functions, e.g.
```
let badCode = '£' + ((startingValue + 10) * 2);
```

File: mandatory/3-function-output.js [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+
+// Gets two words as a parameter and combines them.
+
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+
+  var result = firstWord.concat(" " + secondWord.concat(" " + thirdWord));
```
Comment (ewintram) [inline mandatory/3-function-output.js:19]: remember to use `let` or `const`

**Review by ewintram — COMMENTED** [review]
Amazing work doing all the extra exercises! Great to see you using so many Javascript functions already. Not sure if you saw the messages on Slack about the failing tests in GitHub but there was an error in the GitHub checks in the repo, and all your tests pass locally so nothing to worry about, well done :)

File: exercises/K-functions-parameters/exercise5.js [hunk 3]
```diff
@@ -1,4 +1,8 @@
 // Declare your function here
+function createLongGreeting(name, age){
+
+  return "Hello, my name is " + name + " and I'm " + age + " years old"
+}
 
 const greeting = createLongGreeting("Daniel", 30);
```
[the remark this answers is not in the document]
Comment (Hatice06) [reply exercises/K-functions-parameters/exercise5.js:8]: Thanks for your comment. There is console.log on line9 but I am sure why it was not visible.

File: exercises/K-functions-parameters/exercise3.js [hunk 4]
```diff
@@ -1,5 +1,10 @@
-// Write your function here
+function createGreeting(name){
+  return "Hello, my name is " + name ;
+
```
[the remark this answers is not in the document]
Comment (Hatice06) [reply exercises/K-functions-parameters/exercise3.js:3]: Thanks Busra.

File: extra/1-currency-conversion.js [hunk 5]
```diff
@@ -15,7 +18,9 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(priceGbp) {
+  return parseFloat(((0.99 * priceGbp) * 5.7).toFixed(2));
```
[the remark this answers is not in the document]
Comment (Hatice06) [reply extra/1-currency-conversion.js:22]: Thanks Ellie. I changed as you advised.

**Pull request closed without merging**